microcosm_flask/routing.py

```diff
@@ -29,7 +29,12 @@ def search_foo():
             pass
 
     """
+    graph.use(
```

Comment:
Another graph awkwardness: if `uuid` isn't enforced here, the routes might fail (if they use uuids).